### PR TITLE
[master] add key in return values for create_fcp_template/edit_fcp_template

### PR DESCRIPTION
**Pull request by wngzhe**
Signed-off-by: wngzhe <wngzhe@cn.ibm.com>
1. add key pchids, cpc_sn,cpc_name and lpar to the return value of create_fcp_template/edit_fcp_template
2. pass ut

### CODE REVIEW

**Comment by wngzhe** [comment]
get_pchids_by_fcp_template is fake one ,will rebase and resubmit after code updated

File: zvmsdk/volumeop.py [hunk 1]
```diff
@@ -1032,24 +1032,34 @@ def create_fcp_template(self, name, description: str = '',
                    % (min_fcp_paths_count, len(fcp_devices_by_path)))
             LOG.error(msg)
             raise exception.SDKConflictError(modID='volume', rs=23, msg=msg)
```
Comment (dongyanyang) [inline zvmsdk/volumeop.py:1034]: Update the function comment to reflect the new return values.
please always remember to maintain the comment consistent with code.

File: zvmsdk/volumeop.py [hunk 2]
```diff
@@ -1092,12 +1102,19 @@ def edit_fcp_template(self, fcp_template_id, name=None,
             (fcp_template_id, name, description, fcp_devices,
              host_default, default_sp_list, min_fcp_paths_count)))
         # DML in FCP database
+        zhypinfo = zvmutils.get_zhypinfo()
```
Comment (dongyanyang) [inline zvmsdk/volumeop.py:1105]: Update the function comment to reflect the new return values.

File: zvmsdk/database.py [hunk 3]
```diff
@@ -1426,7 +1430,12 @@ def edit_fcp_template(self, fcp_template_id, name=None, description=None,
                 'storage_providers':
                     [] if tmpl_basic[0]['sp_name'] is None
                     else [r['sp_name'] for r in tmpl_basic],
-                'min_fcp_paths_count': self.get_min_fcp_paths_count(fcp_template_id)
+                'min_fcp_paths_count': self.get_min_fcp_paths_count(fcp_template_id),
```
Comment (dongyanyang) [inline zvmsdk/database.py:1433]: Update the function comment to reflect the new return values.

**Comment by dongyanyang** [comment]
And branch is out-of-date, please rebase to latest master.

**Review by dongyanyang — APPROVED** [review]

**Pull request merged**